### ISSUE TITLE
Can you release the Caption Triplets？

### ISSUE THREAD

**Issue by zetaodu** [Show-han/Zeroshot_REC#1]

**Comment by Show-han** [comment]
I'll upload them this week.

**Comment by Show-han** [comment]
It's on https://drive.google.com/file/d/1iU7RrObmEa_I3hzjxBJxjhEwyGzGhW9n/view?usp=sharing.

**Comment by zetaodu** [comment]
> It's on https://drive.google.com/file/d/1iU7RrObmEa_I3hzjxBJxjhEwyGzGhW9n/view?usp=sharing.

Thanks for your quick reply, I want to know how much resources it takes to train and I can't run it on my single GPU. Thanks~

**Comment by Show-han** [comment]
What do you mean for "train"? You mean fine-tune CLIP model?

**Comment by zetaodu** [comment]
yes！

**Comment by Show-han** [comment]
I fine-tune them on 8*A6000 GPU (48G). You can try to reduce the batchsize, or use the gradient accumulation technique to reduce the computational cost.

**Comment by zetaodu** [comment]
get it, thanks again
